packages/contracts-core/contracts/test/utils/MerkleTest.sol

```diff
@@ -36,34 +38,119 @@ contract MerkleTest is Test {
         return (root, leaf, index, proof);
     }
 
-    // Pulled from String.sol from OpenZeppelin
-    // source: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.6.0/contracts/utils/Strings.sol
-    function toHexString(uint256 value) internal pure returns (string memory) {
-        if (value == 0) {
-            return "0x00";
+    /**
```

Comment:
btw, the more efficient version is in `packages/contracts-bridge/contracts/Encoding.sol` 😅